src/components/FeedPost.tsx

```diff
@@ -0,0 +1,241 @@
+// import axios from "axios";
+import { useEffect, useState } from "react";
+import styled from "styled-components";
+
+import { FeedPostData } from "../MOCKS/homeFeedPage";
+
+const Logo = styled.img`
+  height: 48px;
+  width: 48px;
+  object-fit: cover;
+`;
+
+const UnderlineHover = styled.span`
+  :hover {
+    text-decoration: underline;
+  }
+`;
+
+const StatisticOfTweet = styled.div`
+  :hover {
+    color: rgb(0, 153, 255);
+    transition-duration: 0.2s;
+  }
+`;
+
+const StatisticOfRetweets = styled.div`
+  :hover {
+    color: rgb(41, 228, 166);
+  }
+`;
+const StatisticOfLikes = styled.div`
+  :hover {
+    color: rgb(255, 0, 119);
+  }
+`;
+
+const HoverBackgroundBlue = styled.div`
+  :hover {
+    background: rgb(230, 241, 248);
+    transition-duration: 0.2s;
+  }
+`;
+
+const HoverBackgroundGreen = styled.div`
+  :hover {
+    background: rgb(222, 241, 235);
+    transition-duration: 0.2s;
+  }
+`;
+const HoverBackgroundRed = styled.div`
+  :hover {
+    background: rgb(247, 224, 235);
+    transition-duration: 0.2s;
+  }
+`;
+
+interface TweetPost {
+  id: number;
+  fullName: string;
+  userName: string;
+  userNameId: string;
+  messegeText: string;
+  messegeDate: string;
+  isVerified: boolean;
+  picture: string;
+  userlogo: string;
+  commentCount: number;
+  retweetCount: number;
+  likeCount: number;
+}
+
+const FeedPost = () => {
+  const [posts, setPosts] = useState<TweetPost[]>([]);
+  // const [currentPage, setCurrentPage] = useState(1);
```

Comment:
Remove comments please.

Reply:
![image](https://user-images.githubusercontent.com/94183891/185739566-1f817527-4569-473c-a14a-131a8f038f16.png)

Reply:
Got it.